plugins/foreman_openscap/0.5/index.md

```diff
@@ -76,7 +76,10 @@ Both versions' functionality is the same, the changes only affect to the way it
 
 There are a few components to install:
 
-Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap), foreman_scap_client and puppet-foreman_scap_client.
+* Foreman OpenSCAP (foreman\_openscap)
```

Comment:
I don't see the escaping issue [here](http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#2.Installation) either, but the list looks better.